packages/mobile/src/screens/favorites-screen/AlbumsTab.tsx

```diff
@@ -37,7 +38,7 @@ export const AlbumsTab = () => {
   const isOfflineModeEnabled = useIsOfflineModeEnabled()
```

Comment:
The code in this file and the other file are pretty much identical so working on another PR to decomp them